agent/src/main/java/com/teamscale/jacoco/agent/AgentBase.java

```diff
@@ -96,8 +104,14 @@ protected void initServerEndpoints(Service spark) {
 				Optional.ofNullable(options.getTeamscaleServerOptions().partition).orElse(""));
 		spark.get("/message", (request, response) ->
 				Optional.ofNullable(options.getTeamscaleServerOptions().getMessage()).orElse(""));
+		spark.get("/revision", (request, response) -> this.getRevisionInfo());
+		spark.get("/commit", (request, response) -> Optional.ofNullable(options.getTeamscaleServerOptions().commit)
+				.orElse(new CommitDescriptor("", "")));
```

Comment:
Does Spark automatically call `toString` here on the commit?

Reply:
Yep, looks like it does.